src/detect-lua.c

```diff
@@ -1474,7 +1472,7 @@ static int LuaMatchTest01a(void)
     FLOW_DESTROY(&f);
```

Comment:
If we reach this stage, all `NULL` checks have already been done and we can just straight up clean-up state/release resources.